File: app/src/main/java/com/BlizzardArmory/model/diablo/item/RandomAffix.java
package com.BlizzardArmory.model.diablo.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;


/**
 * The type Random affix.
 */
public class RandomAffix {

    @SerializedName("oneOf")
    @Expose
    private List<OneOf> oneOf;

    /**
     * Gets one of.
     *
     * @return the one of
     */
    public List<OneOf> getOneOf() {
        return oneOf;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/item/RandomAffix.kt
package com.BlizzardArmory.model.diablo.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Random affix.
 */
@Keep
data class RandomAffix(

    @SerializedName("oneOf")

    val oneOf: List<OneOf>
)